NOTICES.txt

```diff
@@ -3,3 +3,8 @@ NUnit 3.0 is based on earlier versions of NUnit, with Portions
 Copyright (c) 2002-2014 Charlie Poole or 
 Copyright (c) 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov or 
 Copyright (c) 2000-2002 Philip A. Craig
+
+Copyright (c) 2008 Jonathan Pryor, Novell
+The file nunitlite/Options.cs is used under MIT license.
+
+Additional third party license notices can be found in THIRD_PARTY_NOTICS.md.
```

Comment:
* NOTICS => NOTICES
* Please make it into a link to the file below

Reply:
Like, an actual web link (ie. `https://github.com/nunit/nunit/blob/master/THIRD_PARTY_NOTICES.md`)?

Reply:
No, a [GitHub relative markdown link](https://github.blog/2013-01-31-relative-links-in-markup-files/)

Reply:
Notices.txt is not a markdown file though, can I change this (rename the file to NOTICES.md)?

Reply:
I would say yes, it gives more flexibility and formatting

Reply:
Yes, agree. Just rename